kodilla-patterns/src/main/java/com/kodilla/patterns/singleton/Logger.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.patterns.singleton;
+
+public class Logger {
```

Comment:
This singleton is not thread safe: see http://www.vogella.com/tutorials/DesignPatternSingleton/article.html#singleton
(In code from article is one small bug, but I hope you manage to find it)
On next meeting I will ask you about correct implementation of singleton, so be prepared!